docs/docs/pages/agent-integration.mdx

```diff
@@ -21,10 +21,16 @@ interface. This adapter pattern allows you to:
 
 ## The AgentAdapter Interface
 
-All agents in Scenario implement the <RefLink link={{ python: "agent_adapter.html#scenario.agent_adapter.AgentAdapter", typescript: "classes/AgentAdapter.html" }} code={{ python: "AgentAdapter", typescript: "AgentAdapter" }} />
-interface with an async <RefLink link={{ python: "agent_adapter.html#scenario.agent_adapter.AgentAdapter.call", typescript: "classes/AgentAdapter.html#call" }} code={{ python: "call", typescript: "call" }} />
-method, which receives an <RefLink link={{ python: "types.html#scenario.types.AgentInput", typescript: "interfaces/AgentInput.html" }} code={{ python: "AgentInput", typescript: "AgentInput" }} />
-and returns <RefLink link={{ python: "types.html#scenario.types.AgentReturnTypes", typescript: "types/AgentReturnTypes.html" }} code={{ python: "AgentReturnTypes", typescript: "AgentReturnTypes" }} />:
+The <RefLink link={{ python: "agent_adapter.html#scenario.agent_adapter.AgentAdapter", typescript: "classes/AgentAdapter.html" }} code={{ python: "AgentAdapter", typescript: "AgentAdapter" }} /> is a simple contract that allows Scenario to work with any agent implementation.
+You only need to provide:
+
+- `role`: The agent’s role in the scenario (`AGENT`, `USER` or `JUDGE`)
```

Comment:
maybe add a (default to `AGENT`) since I think all examples don't specify it right?